be/src/olap/schema_change.cpp

```diff
@@ -1123,6 +1123,9 @@ bool SchemaChangeWithSorting::_internal_sorting(const vector<RowBlock*>& row_blo
     context.partition_id = new_tablet->partition_id();
     context.tablet_schema_hash = new_tablet->schema_hash();
     context.rowset_type = new_rowset_type;
+    if (StorageEngine::instance()->set_default_rowset_type_to_beta()) {
```

Comment:
I think we need a framework to set this rowset_type, instead of setting this at each time the RowsetWriterContext created.
Or it is very easy to miss some places.

Maybe in `RowsetWriterContext`'s constructor?

Reply:
yes, you are right. I will do it in another pr.